public/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
-    <link rel="icon" href="<%= BASE_URL %>favicon.ico" />
+    <link rel="icon" href="./seagate-green.svg" />
```

Comment:
Instead of changing the file, you can rename that file to "favicon.ico" because we use this name "favicon" in general and it avoids confusion.

Reply:
done